R/rownames.R

```diff
@@ -65,6 +67,23 @@ rownames_to_column <- function(df, var = "rowname") {
   new_df
 }
 
+#' @export
+#' @rdname rownames
+rowid_to_column <- function(df, var = "rowid") {
+  stopifnot(is.data.frame(df))
+  
+  if (has_name(df, var))
+    stopc("There is a column named ", var, " already!")
+  
+  if(nrow(df) == 0)
+    stopc('Data frame must have 1 or more rows')
+  
+  rn <- tibble(seq(1, nrow(df)))
+  names(rn) <- var
+  new_df <- cbind(rn, df)
```

Comment:
you mean something like: `add_column(df, !!(var) := seq_len(nrow(df)), .before = 1)`?